cmd/actions/root.go

```diff
@@ -37,8 +37,8 @@ var (
 	flagNamespace string
 )
 
-func mustNewConfigurd(dir, tag string) configurd.Configurd {
-	cnf, err := configurd.New(log, dir, tag)
+func mustNewConfigurd(config configurd.Config) configurd.Configurd {
```

Comment:
Perhaps just `newConfigurd` or call `configurd.New` where needed instead?

Reply:
Must is a standard keyword in go meaning that it will panic if it has an error. e.g. regex.mustCompile